core/services/keeper/registry_synchronizer_core.go

```diff
@@ -61,23 +51,14 @@ type RegistrySynchronizer struct {
 
 // NewRegistrySynchronizer is the constructor of RegistrySynchronizer
 func NewRegistrySynchronizer(opts RegistrySynchronizerOptions) *RegistrySynchronizer {
-	mailRoom := MailRoom{
-		mbUpkeepCanceled:    utils.NewMailbox[log.Broadcast](500),
-		mbSyncRegistry:      utils.NewMailbox[log.Broadcast](1),
-		mbUpkeepPerformed:   utils.NewMailbox[log.Broadcast](3000),
-		mbUpkeepRegistered:  utils.NewMailbox[log.Broadcast](500),
-		mbUpkeepReceived:    utils.NewMailbox[log.Broadcast](500),
-		mbUpkeepMigrated:    utils.NewMailbox[log.Broadcast](500),
-		mbUpkeepGasLimitSet: utils.NewMailbox[log.Broadcast](500),
-	}
 	return &RegistrySynchronizer{
 		chStop:                   make(chan struct{}),
 		registryWrapper:          opts.RegistryWrapper,
 		interval:                 opts.SyncInterval,
 		job:                      opts.Job,
 		jrm:                      opts.JRM,
 		logBroadcaster:           opts.LogBroadcaster,
-		mailRoom:                 mailRoom,
+		mbLogs:                   utils.NewMailbox[log.Broadcast](500),
```

Comment:
Since we're processing logs as they come in, this capacity seems fine. What was the reasoning for the setting though? You should probably add that as a comment in the code or in the git commit message so it's there for the next person to work on this.